_core/regist/regist.php

```diff
@@ -65,17 +65,23 @@ private function insert($info) {
             //'tim'     => $info['local_name'],
             'time'      => $info['time'],
             'pwd'       => $info['post']['password'],
-            'sticky'    => $info['post']['special']['sticky'],
-            'permasage' => $info['post']['special']['permasage'],
-            'locked'    => $info['post']['special']['locked'],
```

Comment:
The columns themselves didn't disappear, but the method to alter them has been changed.